libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -189,6 +189,16 @@ AP_GPS_ERB::_parse_gps(void)
             state.time_week_ms    = _buffer.stat.time;
             state.time_week       = _buffer.stat.week;
         }
+        if (   _buffer.ver.ver_high > 0
+           || (_buffer.ver.ver_high == 0 && _buffer.ver.ver_medium >= 2)) {
```

Comment:
This whole expersion would be valid as ```(   _buffer.ver.ver_high > 0 || _buffer.ver.ver_medium >= 2)``` there is no need to check for ```ver_high == 0``` as a uint8_t must be in order to hit this part of the code.